1. 
Greg was 14, Marcia was 12, Peter was 11, Jan was 10, Bobby was 8, and Cindy was 6 when they started playing the Brady kids on The Brady Bunch. Cousin Oliver was 8 years old when he joined the show. What are the mean, median, and mode of the kids' ages when they first appeared on the show? 

sum(14,12,11,10,8,6,8)=69
len(14,12,11,10,8,6,8)=7

mean= 69/7=9.8
median(14,12,11,10,8,8,6)= 10
mode(14,12,11,10,8,8,6)=8

What are the variance, standard deviation, and standard error?

v = sum((x - mean) ** 2) / n

14-9.8=4.2
12-9.8=2.2
11-9.8=1.2
10-9.8=0.2
8-9.8=-1.8
8-9.8=-1.8
6-9.8=-3.8

4.2**2=17.64
2.2**2=4.84
1.2**2=1.44
0.2**2=0.04
-1.8**2=3.24
-1.8**2=3.24
-3.8**2=14.44

SUM(17.64, 4.34, 1.44, 0.04, 3.24, 3.24, 14.44)=44.38

44.38/7= 6.34

variance= 6.34

standard deviation = variance **0.5

std= 6.34^0.5= 2.51793

sandarderror = s / (n ** 0.5)

se= 2.51793/(7^0.5) = 0.9516

In [6]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import statistics
%matplotlib inline

In [7]:
df = pd.DataFrame()
df['age']=(14,12,11,10,8,6,8)
print("sum =",sum(df['age']))
print("n =",len(df['age']))


print("mean =",np.mean(df['age']))
print("median =",np.median(df['age']))
print("mode =", statistics.mode(df['age']))

print("var =",np.var(df.age))
print("standard deviation =", np.std(df['age']))
print("standard error =", np.std(df['age'] ,ddof=1) / np.sqrt(len(df['age'])))

sum = 69
n = 7
mean = 9.857142857142858
median = 10.0
mode = 8
var = 6.408163547515869
standard deviation = 2.531435012817383
standard error = 1.03345398208


2.
Using these estimates, if you had to choose only one estimate of central tendency and one estimate of variance to describe the data, which would you pick and why?

Answer:
a) I would have chosen an the mean as estimate of central tendency. I did see the distribution of the data , so I knew there are no outlayers, so the median is unnessary.
b)I would have chosen the standard deviation, because it reflects on the heterogeneity the observed data. When using the standard error, the population has a much bigger impact.

3.
Next, Cindy has a birthday. Update your estimates- what changed, and what didn't?
Answer:
Except for median and mode all values will change.

In [8]:
df = pd.DataFrame()
df['age']=(14,12,11,10,8,7,8)
print("sum =",sum(df['age']))
print("n =",len(df['age']))

print("mean =",np.mean(df['age']))
print("median =",np.median(df['age']))
print("mode =", statistics.mode(df['age']))

print("var =",np.var(df.age))
print("standard deviation =", np.std(df['age']))
print("standard error =", np.std(df['age'] ,ddof=1) / np.sqrt(len(df['age'])))

sum = 70
n = 7
mean = 10.0
median = 10.0
mode = 8
var = 5.4285712242126465
standard deviation = 2.3299293518066406
standard error = 0.951189768122


4.
Nobody likes Cousin Oliver. Maybe the network should have used an even younger actor. Replace Cousin Oliver with 1-year-old Jessica, then recalculate again. Does this change your choice of central tendency or variance estimation methods?

Answer:
This does change the results again. Using mode, does not make any sense anymore, because each value will occure an equal number of times. But all other methods are still fine to use.

In [10]:
df = pd.DataFrame()
df['age']=(14,12,11,10,8,6,1)
print("sum =",sum(df['age']))
print("n =",len(df['age']))

print("mean =",np.mean(df['age']))
print("median =",np.median(df['age']))
#print("mode =", statistics.mode(df['age'])) # returns stats error, because all value occure an equal number of times

print("var =",np.var(df.age))
print("standard deviation =", np.std(df['age']))
print("standard error =", np.std(df['age'] ,ddof=1) / np.sqrt(len(df['age'])))

sum = 62
n = 7
mean = 8.857142857142858
median = 10.0
var = 16.122447967529297
standard deviation = 4.015276908874512
standard error = 1.63922989657


5.
On the 50th anniversary of The Brady Bunch, four different magazines asked their readers whether they were fans of the show. The answers were: TV Guide 20% fans Entertainment Weekly 23% fans Pop Culture Today 17% fans SciPhi Phanatic 5% fans

Based on these numbers, what percentage of adult Americans would you estimate were Brady Bunch fans on the 50th anniversary of the show?

In [13]:
df = pd.DataFrame()
df['fans']= (0.20,0.23,0.17,0.05)

np.mean(df['fans'])

0.16250000000000003

Answer:
The avarage amount of readers of those magazines who liked The Brady Bunch is 16%.